src/utils/api.js

```diff
@@ -12,6 +12,12 @@ const octoRequest = request.defaults({
   },
 });
 
+/**
+ *
+ * @param {Object} param
+ * @param {String} param.username
```

Comment:
should there be two params here?

Reply:
yes, the second one is to describe the shape of the param because were doing destructuring